src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -1,249 +1,194 @@
-            </div>
-          </div>
-        </div>
-
-        <!-- Force change password -->
-        <div class="form-group row"
-             *ngIf="!isCurrentUser() && !authStorageService.isSSO()">
-          <div class="cd-col-form-offset">
-            <div class="custom-control custom-checkbox">
-              <input type="checkbox"
-                     class="custom-control-input"
-                     id="pwdUpdateRequired"
-                     name="pwdUpdateRequired"
-                     formControlName="pwdUpdateRequired">
-              <label class="custom-control-label"
-                     for="pwdUpdateRequired"
-                     i18n>User must change password at next logon</label>
-            </div>
-          </div>
-        </div>
+           class="form-header">{{ action | titlecase }} {{ resource | upperFirst }}
+      </div>
+      <!-- UserName -->
+      <div class="form-item">
+        <cds-text-label for="username"
+                        cdRequiredField="Username"
+                        [invalid]="!userForm.controls.username.valid && userForm.controls.username.dirty"
+                        [invalidText]="usernameError"
+                        i18n>Username
+          <input cdsText
+                 placeholder="Username..."
+                 i18n-placeholder
+                 id="username"
+                 name="username"
+                 formControlName="username"
+                 [invalid]="!userForm.controls.username.valid && userForm.controls.username.dirty"
+                 autofocus>
+        </cds-text-label>
+        <ng-template #usernameError>
+          <span *ngIf="userForm.showError('username', formDir, 'required')">
+            <ng-container i18n>
+             This field is required.
+            </ng-container>
+          </span>
+          <span *ngIf="userForm.showError('username', formDir, 'notUnique')">
+            <ng-container i18n>
+              The username already exists.
+            </ng-container>
+          </span>
+        </ng-template>
+      </div>
+      <!-- Password -->
+      <div class="form-item">
+        <cds-password-label for="password"
+                            label="Password..."
+                            [helperText]="helperText"
+                            [invalid]="!userForm.controls.password.valid && userForm.controls.password.dirty"
+                            [invalidText]="passwordError"
+                            i18n>Password
+        <input cdsPassword
+               type="password"
+               placeholder="Password..."
+               id="password"
+               name="password"
+               autocomplete="new-password"
+               formControlName="password"
+               >
+        </cds-password-label>
+        <ng-template #helperText>
+          <span *ngIf="!passwordValuation">{{passwordPolicyHelpText}}</span>
+        </ng-template>
+        <ng-template #passwordError>
```

Comment:
re-iterating back here, let's remove the `helperText` input and keep the existing `cd-helper`. This information doesn't need to show up everytime, so its better to have it under the helper icon and they can just view it whenever the want to by hovering over the icon.
```suggestion
        <cd-helper *ngIf="passwordPolicyHelpText.length > 0"
                       class="text-pre-wrap"
                       html="{{ passwordPolicyHelpText }}">
        </cd-helper>
        <ng-template #passwordError>
```

Reply:
![image](https://github.com/user-attachments/assets/c54fd3a9-59f5-414b-8591-87f2b496a81a)